pyfstat/core.py

```diff
@@ -1076,10 +1077,11 @@ def plot_twoF_cumulative(
         label, outdir : str
         add_pfs : bool
             If true, plot the predicted 2F and standard deviation
+        injection_parameters: dict
+            Dictionary with injection parameters to predict Fstat
+            (h0, cosi, psi, Alpha, Delta, F0)
         N : int
-            Number of points to use
-        injectSources : dict
-            See `ComputeFstat`
+            Number of points to use for PredictFStat.
```

Comment:
If we're being backwards-incompatible with arguments anyway, we should call this `N_PFS` or something and also introduce here an explicit `N_CFS` that gets passed to `calculate_twoF_cumulative()`.

Reply:
Agree, but this is a Python code: `predict_fstat_segments` sounds clearer ?